common/services/prismic/types/index.ts

```diff
@@ -64,8 +64,10 @@ export function isFilledLinkToMediaField(
   );
 }
 
-export function isFilledSliceZone(
-  sliceZone: prismic.SliceZone
-): sliceZone is prismic.SliceZone<prismic.SharedSlice, 'filled'> {
+export function isFilledSliceZone<
+  T extends prismic.SharedSlice = prismic.SharedSlice,
+>(
+  sliceZone: prismic.SliceZone<T>
+): sliceZone is prismic.SliceZone<T, 'filled'> {
```

Comment:
[This](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/pages/guides/exhibitions/%5Bid%5D/%5Btype%5D/%5Bstop%5D/index.tsx#L180) was flagging as "never" by TS, so I added this to make sure it was typed properly when used.